build.gradle.kts

```diff
@@ -221,36 +245,44 @@ gradle.projectsEvaluated {
 
 // --- Intellij plugin
 
-intellij {
-  version.set(ideaVersion)
-  type.set(ideaType)
-  pluginName.set("IdeaVim")
-
-  updateSinceUntilBuild.set(false)
+intellijPlatform {
+  pluginConfiguration {
+    name = "IdeaVim"
+    changeNotes.set(
+      """<a href="https://youtrack.jetbrains.com/issues/VIM?q=State:%20Fixed%20Fix%20versions:%20${version.get()}">Changelog</a>"""
+    )
 
-  downloadSources.set(downloadIdeaSources.toBoolean())
-  instrumentCode.set(instrumentPluginCode.toBoolean())
-  intellijRepository.set("https://www.jetbrains.com/intellij-repository")
-  plugins.set(listOf("AceJump:3.8.11"))
-}
+    ideaVersion {
+      // Set the since-build value, but leave until-build open ended (default is MAJOR.*)
+      // Don't forget to update plugin.xml
+      // TODO: Do we need this to be here *and* in plugin.xml?
+      sinceBuild.set("233.11799.67")
+      untilBuild.set(provider { null })
+    }
```

Comment:
If we set `since-build` here, will the Plugin DevKit use it for analysis? We have it duplicated in `plugin.xml` right now, it would be nice to only set in one place

Reply:
You can set it in Gradle build file only. Those values are used by the `patchPluginXml` file to update the `plugin.xml` file. All further checks performed by the Gradle plugin respect those values.

Reply:
What about the DevKit though? Does it look at the patched or unpatched plugin.xml?

Reply:
After some tests, it seems that DevKit only uses the value in the unpatched `plugin.xml`, so it's still worth keeping both values, even though it would make things easier to specify only in the build scripts

Reply:
Resolved: Let the Gradle plugin handle patching the `since-build` value. This value now comes solely from the target IDE dependency. It is no longer specified in `plugin.xml` or `build.gradle.kts`. The format is slightly different - `{branch}.{build}` instead of `{branch}.{build}.{minor}`. This is the default for the Gradle plugin; the minor version can be different between IDEs released from the same branch.